torch/csrc/jit_decomp_interface.cpp

```diff
@@ -0,0 +1,23 @@
+#include <torch/csrc/jit_decomp_interface.h>
+
+namespace torch {
+namespace autograd {
+namespace impl {
+
+namespace {
+JitDecompInterface* fns = nullptr;
+}
+
+void setJitDecompInterface(JitDecompInterface* f) {
+  fns = f;
+}
+JitDecompInterface* getJitDecomp() {
+  TORCH_CHECK(
+      fns,
+      "Support for JIT decompositions has not been loaded; have you linked against TBD?")
```

Comment:
This message should be more informative. I'll probably update this and other things once I'm sure internal builds pass.